### PR TITLE
Update soul scream timer to match the data from logs

### PR DESCRIPTION
**Pull request by kuskmen**
It seems that soul scream is happening every 11.3s consistently, I don't know much about your timers and if they support floating numbers but it seems that 10 seconds are not what it is according to logs.

https://classic.warcraftlogs.com/reports/vGmV2z1cbNQhA7xX/#fight=last&hostility=1&source=196&type=casts&ability=41545&view=events

### CODE REVIEW

**Comment by MysticalOS** [comment]
In that log i already see some that were less than 11.3, DBM uses the lowest seen time not the average. that log alone has 1 that's just 11.0 so i definitely wouldn't use 11.3, at very least I'd consider changing it to 11 if further logs didn't show it going even lower.

I want to say it is a 10 second timer but it's rarely 10 since there is almost always a delay before recast. in fast that's true of most timers. I'll often have a timer like 36 and then one day i'll see the cast come at 35 and realize it was prob 35 all along in scripting but just rarely cast without delay. but in that case I'd set it too 35 as well because i'm still of mind that I'd father the timer be 1 second too fast 90% of the time, than 1 second too slow 10% of time. People need to look at DBM and KNOW that if timer is still on screen, they are safe from ability, it should never come before expected, but it's ok to come slightly after expected.

TL/DR, i want to look at more logs and see if I can find any cases below 11.0, because it's till possible 11 is lowest time and 10 (from retail) doesn't match classic. But if I do find anything under 11 then it's probably safe bet it is same and i probably won't change it.

**Comment by kuskmen** [comment]
It cant be lower than 11 because as far as I understood boss script is running every 1.62 seconds and 10 second is the cooldown of the scream which means 11-11.3 will be the lowest multiple of 1.62. This is as far as I understood, couple of guys from Fight Club discord confirmed log timers tho I do agree that setting the timer to 11 would be better than exactly 11.3.

**Review by MysticalOS — APPROVED** [review]
